### ISSUE TITLE
Certain Quest 2 bindings don't work, such as Open PDA (click Y)

### ISSUE THREAD

**Issue by NibblyPig** [Okabintaro/SubmersedVR#154]
Found this issue when I pressed literally every single button on both controllers, but couldn't open the PDA.

In the bindings page, it is bound to click Y, however it seems this binding doesn't function.

Certain other bindings also don't work either.

Workaround: I was able to rebind it to touch Y instead of click Y, which isn't ideal but it does work. 

As the game progresses I believe I will need certain other button presses too, may have to get creative.

**Comment by Okabintaro** [comment]
Have you tried reverting to the default bindings in steamvr?
If you played the game without the mod before and changed some bindings this can happen I think.

**Comment by NibblyPig** [comment]
Yup, tried reverting various other suggested tweaks but it just won't accept that input, plus a couple of others.

**Comment by Okabintaro** [comment]
What streamer and  game version do you use? I just tested this last weekend and it worked fine with Airlink + Virtual Desktop.
See: https://github.com/Okabintaro/SubmersedVR?tab=readme-ov-file#the-game-doesnt-work-what-could-be-wrong

**Comment by Okabintaro** [comment]
No further response, I had no issues with the same setup.
